arangod/RocksDBEngine/RocksDBRestReplicationHandler.cpp

```diff
@@ -436,15 +447,23 @@ void RocksDBRestReplicationHandler::handleCommandCreateKeys() {
 
   // bind collection to context - will initialize iterator
   Result res;
-  DataSourceId cid;
   uint64_t numDocs;
+  DataSourceId cid;
   std::tie(res, cid, numDocs) = ctx->bindCollectionIncremental(_vocbase, collection);
-
   if (res.fail()) {
     generateError(res);
     return;
   }
 
+  if (numDocs > 1000000 && quick == "true") {
```

Comment:
Should `_quickKeysLimit` be used here instead of the hard-coded value?
```suggestion
  if (numDocs > 1000000 && quick == "true") {
```